optaplanner-core/src/main/java/org/optaplanner/core/api/score/AbstractScore.java

```diff
@@ -178,11 +177,6 @@ public boolean isSolutionInitialized() {
         return initScore >= 0;
     }
 
-    @Override
-    public boolean isCompatibleArithmeticArgument(Score otherScore) {
```

Comment:
I 'll add something in upgrade recipe from 6.x to 7.x about this, in the exotic case that a user has defined a custom score.